docs/src/components/sidebarIndex.js

```diff
@@ -2,14 +2,14 @@
 
 export type sidebarIndexType = Array<{|
   sectionPathname: string,
-  displayName: string,
-  pages?: Array<string>,
-  subsections?: Array<{| [string]: Array<string> |}>,
+    displayName: string,
+      pages ?: Array < string >,
+      subsections ?: Array < {| [string]: Array<string> |}>,
```

Comment:
My editor kept doing this when I saved. I'll update when I address feedback and make sure to remove these changes